src/controllers/cartItemController.ts

```diff
@@ -48,6 +50,71 @@ export const deleteCartItemController = async (req: Request, res: Response) => {
         }else{
             res.status(500).json({ message: "An error occurred while deleting the cart item" });
         }
+    }
+}
+
+export const deleteAllCartItems = async (req: Request, res: Response) => {
+    try{
+        const { userId } = req.params;
+        await deleteAllCartItemsForUserService(userId);
```

Comment:
I think this should return something, true/false mabe then check if true return the result 200